src/frontend/src/routes/login/index.tsx

```diff
@@ -0,0 +1,30 @@
+import Footer from "~/components/Footer";
+import AppLink from "~/elements/AppLink";
+import GithubIconSrc from "./icons/github.png";
```

Comment:
Können wir die Icons gleich in den public ordner legen? Sonst befürchte ich, dass man sie nicht findet, wenn man sie an anderer Stelle benötigt.